library/src/main/java/com/novoda/downloadmanager/RoomDownloadsPersistence.java

```diff
@@ -139,4 +139,46 @@ public boolean update(DownloadBatchId downloadBatchId, boolean notificationSeen)
         database.roomBatchDao().update(roomBatch);
         return true;
     }
+
+    @Override
+    public void persistCompletedBatch(CompletedDownloadBatch completedDownloadBatch) {
+        DownloadBatchId downloadBatchId = completedDownloadBatch.downloadBatchId();
+        DownloadBatchTitle downloadBatchTitle = completedDownloadBatch.downloadBatchTitle();
+        DownloadBatchStatus.Status downloadBatchStatus = DownloadBatchStatus.Status.DOWNLOADED;
+        long downloadedDateTimeInMillis = completedDownloadBatch.downloadedDateTimeInMillis();
+
+        DownloadsBatchPersisted persistedBatch = new LiteDownloadsBatchPersisted(
+                downloadBatchTitle,
+                downloadBatchId,
+                downloadBatchStatus,
+                downloadedDateTimeInMillis,
+                true
```

Comment:
what is `true`?

Reply:
it's the notification 😂 I hate adding these now because the IDE shows what the param is anyway, it's just GH doesn't. 599d855